tests-ng/test_capabilities.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+import pytest
+
+from plugins.capabilities import Capabilities
+
+# There are two debian packages that provide an 'arping' binary
+# https://packages.debian.org/search?searchon=contents&keywords=arping&mode=path&suite=stable&arch=any
+# /usr/bin/arping is provided by iputils-arping
+# /usr/sbin/arping is provided by arping
+# We only use iputils-arping
+capabilities_allowlist = set(["/usr/bin/arping cap_net_raw=ep"])
```

Comment:
*Nitpick* (feel free to ignore): Instead of casting a list here, it's possible to use the direct set expression here as the set is not empty:

```suggestion
capabilities_allowlist = {"/usr/bin/arping cap_net_raw=ep"}
```